test/libsolidity/syntaxTests/types/mapping/assignment_local_err.sol

```diff
@@ -0,0 +1,20 @@
+contract C {
+    mapping (uint => uint) a;
+    mapping (uint => int) b = a;
+    function foo() public view {
+        mapping (uint => int) storage c = a;
+    }
+}
+
+contract D {
+    mapping (uint => uint) a;
+    mapping (uint => uint) b;
+    function foo() public view {
+        mapping (uint => uint) storage c = a;
+        b = c;
+    }
+}
+// ----
+// TypeError: (47-74): Mappings cannot be assigned to.
+// TypeError: (117-152): Type mapping(uint256 => uint256) is not implicitly convertible to expected type mapping(uint256 => int256).
```

Comment:
This is actually testing a different feature, right? Can you move it into its own file?

Reply:
Yes, I will move it.